src/renderers/common/RenderObject.js

```diff
@@ -214,6 +219,10 @@ export default class RenderObject {
 
 					valueKey += '}';
 
+				} else {
+
+					valueKey = String( value );
```

Comment:
@sunag This additional branch is required if `value` is not `null` but also not `number` or `object` (e.g. `boolean`). Previously the value was ignored now it is honored in the cache key.